docs/api-reference/next.config.js/rewrites.md

```diff
@@ -300,10 +300,48 @@ module.exports = {
   <summary><b>Examples</b></summary>
   <ul>
     <li><a href="https://github.com/vercel/next.js/tree/canary/examples/custom-routes-proxying">Incremental adoption of Next.js</a></li>
+    <li><a href="https://github.com/vercel/next.js/tree/canary/examples/with-zones">Using Multiple Zones</a></li>
   </ul>
 </details>
 
-Rewrites allow you to rewrite to an external url. This is especially useful for incrementally adopting Next.js.
+Rewrites allow you to rewrite to an external url. This is especially useful for incrementally adopting Next.js. The following is an example rewrite for redirecting the `/blog` route of your main app to an external site.
+
+```js
+module.exports = {
+  async rewrites() {
+    return [
+      {
+        source: '/blog',
+        destination: 'https://example.com/blog',
+      },
+      {
+        source: '/blog/:slug',
+        destination: 'https://example.com/blog/:slug', // Matched parameters can be used in the destination
+      },
+    ]
+  },
+}
+```
+
+If you're using `trailingSlash: true`, you also need to insert a trailing slash in the redirects.
+
+```js
+module.exports = {
+  trailingSlash: 'true',
+  async rewrites() {
+    return [
+      {
+        source: '/blog/',
+        destination: 'https://example.com/blog/',
+      },
+      {
+        source: '/blog/:path*',
+        destination: 'https://example.com/blog/:path*',
+      },
+    ]
+  },
+}
+```
 
 ```js
 module.exports = {
```

Comment:
Should this be removed?

Reply:
Yes! Good catch, thank you.

Reply:
Lmk when ready for review!

Reply:
Will do! I'm going to wait until the `trailingSlash` bug is fixed since what I added doesn't completely work right now.

Reply:
We resolved the issue! PR is now ready for review.